Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -980,6 +980,13 @@ extension BrowserViewController: LongPressGestureDelegate {
         actionSheetController.title = dialogTitleURL!.absoluteString
         var cancelAction = UIAlertAction(title: "Cancel", style: UIAlertActionStyle.Cancel, nil)
         actionSheetController.addAction(cancelAction)
+
+        if let popoverPresentationController = actionSheetController.popoverPresentationController {
```

Comment:
Can we add a UITest that verifies this is at least reasonable?  We have a range of testers with devices that can ensure this doesn't regress.  In addition, we'll want to test long-press event details eventually.